controllers/ca/ca_controller.go

```diff
@@ -492,6 +493,18 @@ func GetConfig(conf *hlfv1alpha1.FabricCA, client *kubernetes.Clientset, chartNa
 	if spec.Istio != nil && len(spec.Istio.Hosts) > 0 {
 		istioHosts = spec.Istio.Hosts
 	}
+	gatewayApiPort := 443
+	if spec.GatewayApi != nil && spec.GatewayApi.Port != 0 {
+		gatewayApiPort = spec.GatewayApi.Port
+	}
+	gatewayApiHosts := []string{}
+	if spec.GatewayApi != nil && len(spec.GatewayApi.Hosts) > 0 {
+		gatewayApiHosts = spec.GatewayApi.Hosts
```

Comment:
I would only assign the hosts when the gatewayAPIPort and gatewayAPIClassname are set. Since templates is checking only the hosts to create the gateway CRDs.

Reply:
Yes @dviejokfs , noted! Will update it.